server/http/helloapp/embedded_wish.go

```diff
@@ -0,0 +1,197 @@
+package helloapp
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (h *helloapp) handleCreateEmbedded(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, data *api.Call) (int, error) {
+	err := h.asBot(func(mmclient *model.Client4, botUserID string) error {
+		post := &model.Post{
+			Message:   "Debug form",
+			ChannelId: data.Context.ChannelID,
+			UserId:    botUserID,
+		}
+		post.AddProp("appID", AppID)
```

Comment:
constants.AppID

Reply:
A hypothetical question, can we envision multiple embedded forms added to a post? Should AppID be embedded in the Form?

Either way I don't like having `AppID` as a top-level prop, because it relates that a post "belongs to" an App, which should not be the case.

Reply:
I do not like much the idea of several plugins displaying a dialog in the same message. Also, I think the post does indeed belong to the app, since it is the one posting it as a form, and will be the one modifying it.

Reply:
0/5 log as a separate issue to discuss/resolve later?